CSC8503/GameTechRenderer.cpp

```diff
@@ -304,49 +304,50 @@ void GameTechRenderer::RenderCamera() {
 }
 
 void GameTechRenderer::DrawWallsFloorsInstanced(Matrix4& viewMatrix, Matrix4& projMatrix) {
+	for (int i = 0; i < 3; i++) {
```

Comment:
Nice to have: adding a constant variable will be nice bcs I can't understand why are we looping for 3 times at the first look ✌️ 

```suggestion
	for (int i = 0; i < SOME_CONSTANT_VARIABLE; i++) {
```